neqo-crypto/src/cert.rs

```diff
@@ -38,40 +29,6 @@ fn peer_certificate_chain(fd: *mut PRFileDesc) -> Option<ItemArray> {
     }
 }
 
-// As explained in rfc6961, an OCSPResponseList can have at most
-// 2^24 items. Casting its length is therefore safe even on 32 bits targets.
-fn stapled_ocsp_responses(fd: *mut PRFileDesc) -> Option<Vec<Vec<u8>>> {
-    let ocsp_nss = unsafe { SSL_PeerStapledOCSPResponses(fd) };
-    match NonNull::new(ocsp_nss as *mut SECItemArray) {
-        Some(ocsp_ptr) => {
-            let mut ocsp_helper: Vec<Vec<u8>> = Vec::new();
-            let Ok(len) = isize::try_from(unsafe { ocsp_ptr.as_ref().len }) else {
-                qerror!("[{fd:p}] Received illegal OSCP length");
-                return None;
-            };
-            for idx in 0..len {
-                let itemp: *const SECItem = unsafe { ocsp_ptr.as_ref().items.offset(idx).cast() };
-                let item = unsafe { null_safe_slice((*itemp).data, (*itemp).len) };
-                ocsp_helper.push(item.to_owned());
-            }
-            Some(ocsp_helper)
-        }
-        None => None,
-    }
-}
-
-fn signed_cert_timestamp(fd: *mut PRFileDesc) -> Option<Vec<u8>> {
```

Comment:
Not having this in Firefox might not be OK.  We're rolling out CT and I thought we needed this for that.